### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#189] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253126-29d55c0e-7c3b-4db1-83c4-b1f88bc4885a.png)
***
_L'interpretazione più corretta sembrerebbe quella che si tratti di elementi distinti (essendo che non è specificato, come avviene invece in altri esercizi, che gli elementi sono identici)_

1. Soluzione che li considera **distinti**:
	https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/189#discussioncomment-10286351

2. Soluzione che li considera **identici**:
	https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/189#discussioncomment-10286350

**Comment by Exyss** [comment]
Proposta di soluzione nel caso si considerino gli **N** elementi come **identici** tra loro (e quindi che non conta le diverse possibili combinazioni di elementi nei due sottoinsiemi)
***
Effettuo una traduzione del problema: distribuisco N pallini sulle due colonne

| Gruppo 1 | Gruppo 2 |
| ------------ | ------------- | 


Tuttavia, per divide in due gruppi è necessario che ogni gruppo abbia almeno 1 elemento, dunque sottraggo 2 pallini dal totale, in modo che siano già distribuiti tra i due gruppi.

La situazione è ora:

| Gruppo 1 | Gruppo 2 |
| :----------: | :------------: | 
|       •        |         •        | 

A questo punto quindi dobbiamo distribuire gli N-2 pallini rimanenti su 2 gruppi, dunque la soluzione è

$\binom{N-2+2-1}{2-1} = \binom{N-1}{1} = N-1$

**Comment by D0ppiaP-02** [comment]
Proposta di soluzione nel caso si considerino gli **n** elementi come **distinti** tra loro (e quindi che conta anche le diverse possibili combinazioni di elementi nei due sottoinsiemi)
***
$\frac{2^n - 2}{2}$

Dove $2^n - 2$ rappresenta il numero di sottinsiemi possibili escludendo quello vuoto e l'insieme stesso. Ognuno di questi implica anche il sottinsieme complemento del primo e quindi si divide per due per non contare due volte le stesse configurazioni con i sottinsiemi cambiati di posto.
